src/domain/stake.ts

```diff
@@ -19,13 +19,15 @@ export function insertStake(
   staker: Address,
   amount: BigInt,
   proposalId: string,
+  daoId: string,
```

Comment:
why it is daoId and not dao ?

Reply:
Same reason the above is proposalId and not proposal. You don't need the object just the ID.

Reply:
why not the object ?